ogm/roadmap.adoc

```diff
@@ -49,14 +29,10 @@ Clean query parser design::
   to find the type of a property for HQL literal conversion
 ** extract this logic and reuse it for all backends
 
```

Comment:
I think there are more detailed than need in the 'Clean query parser design' session.

I would change it to something like:
```
Query parser:
Improve property type detection in queries.
```